source/Halibut/Transport/SecureListener.cs

```diff
@@ -65,12 +66,27 @@ public SecureListener(IPEndPoint endPoint, X509Certificate2 serverCertificate, F
             this.serverCertificate = serverCertificate;
             this.protocolHandler = protocolHandler;
```

Comment:
It’d be better for the constructor in line 63 to call your new constructor - will reduce duplication

Reply:
Done